### PR TITLE
Chore: Missing some English translation keywords

### PR DESCRIPTION
**Pull request by Karting06**
## Proposed changes (including videos or screenshots)

Add missing translation keys in `en.i18n.json` to be able to translate them via Lingohub.

## Issue(s)

Those translation keys were missing from the English JSON which led to untranslated strings in other languages. 

## Steps to test or reproduce

Use Rocket.Chat in another language and notice the missing translations.

### CODE REVIEW

File: packages/rocketchat-i18n/i18n/en.i18n.json [hunk 1]
```diff
@@ -4168,5 +4204,6 @@
   "Your_server_link": "Your server link",
   "Your_temporary_password_is_password": "Your temporary password is <strong>[password]</strong>.",
   "Your_TOTP_has_been_reset": "Your Two Factor TOTP has been reset.",
-  "Your_workspace_is_ready": "Your workspace is ready to use 🎉"
+  "Your_workspace_is_ready": "Your workspace is ready to use 🎉",
+  "Zapier": "Zapier"
```
Comment (sampaiodiego) [inline packages/rocketchat-i18n/i18n/en.i18n.json:4208]: Zapier is the name of a company/product, it shouldn't be translated.

File: packages/rocketchat-i18n/i18n/en.i18n.json [hunk 2]
```diff
@@ -1140,6 +1146,7 @@
   "CROWD_Reject_Unauthorized": "Reject Unauthorized",
   "Crowd_Remove_Orphaned_Users": "Remove Orphaned Users",
   "Crowd_sync_interval_Description": "The interval between synchronizations. Example `every 24 hours` or `on the first day of the week`, more examples at [Cron Text Parser](http://bunkat.github.io/later/parsers.html#text)",
+  "Curl": "Curl",
```
Comment (sampaiodiego) [inline packages/rocketchat-i18n/i18n/en.i18n.json:1149]: `curl` is a command, it shouldn't be translated.

File: packages/rocketchat-i18n/i18n/en.i18n.json [hunk 3]
```diff
@@ -1887,8 +1904,10 @@
   "How_responsive_was_the_chat_agent": "How responsive was the chat agent?",
   "How_satisfied_were_you_with_this_chat": "How satisfied were you with this chat?",
   "How_to_handle_open_sessions_when_agent_goes_offline": "How to Handle Open Sessions When Agent Goes Offline",
+  "HTML": "HTML",
```
Comment (sampaiodiego) [inline packages/rocketchat-i18n/i18n/en.i18n.json:1907]: `HTML` is a markup language and should not be translated

File: packages/rocketchat-i18n/i18n/en.i18n.json [hunk 4]
```diff
@@ -2123,6 +2145,7 @@
   "join-without-join-code_description": "Permission to bypass the join code in channels with join code enabled",
   "Joined": "Joined",
   "Joined_at": "Joined at",
+  "JSON": "JSON",
```
Comment (sampaiodiego) [inline packages/rocketchat-i18n/i18n/en.i18n.json:2148]: `JSON` stands for JavaScript Object Notation and should not be translated.

File: packages/rocketchat-i18n/i18n/en.i18n.json [hunk 5]
```diff
@@ -2856,6 +2881,7 @@
   "Opened": "Opened",
   "Opened_in_a_new_window": "Opened in a new window.",
   "Opens_a_channel_group_or_direct_message": "Opens a channel, group or direct message",
+  "OpLog": "OpLog",
```
Comment (sampaiodiego) [inline packages/rocketchat-i18n/i18n/en.i18n.json:2884]: `oplog` is MongoDB feature and shouldn't be translated.

File: packages/rocketchat-i18n/i18n/en.i18n.json [hunk 6]
```diff
@@ -1887,8 +1904,10 @@
   "How_responsive_was_the_chat_agent": "How responsive was the chat agent?",
   "How_satisfied_were_you_with_this_chat": "How satisfied were you with this chat?",
   "How_to_handle_open_sessions_when_agent_goes_offline": "How to Handle Open Sessions When Agent Goes Offline",
+  "HTML": "HTML",
   "I_ll_do_it_later": "I'll do it later",
   "I_saved_my_password_close_this_message": "I saved my password, close this message",
+  "ID": "ID",
```
Comment (sampaiodiego) [inline packages/rocketchat-i18n/i18n/en.i18n.json:1910]: `ID` is the contraction for "identifier", not sure it should be translated.

File: packages/rocketchat-i18n/i18n/en.i18n.json [hunk 7]
```diff
@@ -2906,6 +2934,7 @@
   "Personal_Access_Tokens": "Personal Access Tokens",
   "Phone": "Phone",
   "Phone_number": "Phone number",
+  "PID": "PID",
```
Comment (sampaiodiego) [inline packages/rocketchat-i18n/i18n/en.i18n.json:2937]: `PID` stands for "process ID" .. not sure it should be translated.

**Review by sampaiodiego — CHANGES_REQUESTED** [review]
thanks @Karting06 for your contribution.. most of the terms shouldn't be translated because they're actually names and not words.

@ggazzo I saw most of those missing keywords were recently added during the React rewrite. can you please make sure to always add the keys to the English translation files as well? thx

**Comment by Karting06** [comment]
Yeah, I know those terms. 
@sampaiodiego, I added them because I found similar terms in the list (`SMTP`, `SAML`). 
I will change it then.

**Comment by Karting06** [comment]
Hi @sampaiodiego,
I don't know how to change the `change requested` status. I committed the new changes. 
Should I do somethinge else?

**Pull request merged**